Lifestyle/digital-zeitgebers.30m.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+# <bitbar.title>Digital Zeitgebers</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Edward Qiu</bitbar.author>
+# <bitbar.author.github>edwardqiu</bitbar.author.github>
+# <bitbar.desc>A fuzzy clock that reminds you of certain sleep hyigene recommendations, such as when to avoid blue light, caffeine, etc., at the proper time.</bitbar.desc>
+# <bitbar.image>https://github.com/edwardqiu/digital-zeitgebers/blob/master/docs/banner.png</bitbar.image>
```

Comment:
Your image link returns HTML not an image directly.  You probably want https://raw.githubusercontent.com/edwardqiu/digital-zeitgebers/master/docs/banner.png